### PR TITLE
set up associations

### PR DESCRIPTION
**Pull request by camilevahviraki**
### **In this pull request we did the following:**

- Created a class Classroom
- Created the has-many/belongs-to relationship between Classroom and Student
- Created a class Book
- Created a class Rental
- Created the many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental

### CODE REVIEW

File: classroom.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  def add_rental(date)
+    new.Rental(date)
+  end
+end
+
+class Rental
+  def initialize(date, person_obj, book_obj)
+    @date = date
+    @person_data = person_obj
```
Comment (rotimiazeez) [inline classroom.rb]: - Kindly make sure the rental is adding itself to the @person_data achieving the 'belongs-to' association 👍🏽
Hint:
```
`person.rentals << self`
```

File: classroom.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    @book_data = book_obj
```
Comment (rotimiazeez) [inline classroom.rb]: - Kindly make sure the rental is adding itself to the @book_data achieving the 'belongs-to' association 👍🏽
Hint:
```
`book_data<< self`
```

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @camilevahviraki 👏🏽, 
## Good job so far! 👍🏽 
### Summary ♨️ 
- No Linter error ✔️ 
- Good and descriptive PR 💯 
- Correct github flow
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
Hi @camilevahviraki,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**